pkg/driver/microphone/microphone.go

```diff
@@ -111,6 +111,8 @@ func (m *microphone) AudioRecord(inputProp prop.Media) (audio.Reader, error) {
 	config.PerformanceProfile = malgo.LowLatency
 	config.Capture.Channels = uint32(inputProp.ChannelCount)
 	config.SampleRate = uint32(inputProp.SampleRate)
+	//FIXME: Turn on the microphone with the current device id
```

Comment:
Is it not fixed yet and to be fixed in future? (FIXME?)

Reply:
It has been modified，